### DISCUSSION TITLE
How long does it take for npmjs.com packages to show at linkedsoftwaredependencies.org ?

### DISCUSSION THREAD

**Discussion by ixuz** [LinkedSoftwareDependencies/Components.js#71] [Q&A] [answered]
Hi there!

I've recently published a new package to npmjs.com and I wonder how long is does it take for it to show up at https://linkedsoftwaredependencies.org/ ?

If I'm not mistaken, there should be some information at this link: https://linkedsoftwaredependencies.org/bundles/npm/@iotakingdoms/common

This is the repo if it helps: https://github.com/iotakingdoms/common
And here's the packages published to npmjs: https://www.npmjs.com/package/@iotakingdoms/common

Thanks! :slightly_smiling_face:

**Comment by rubensworks** [comment, marked as answer]
Hmm, that should be quite instant.

@joachimvh could there be a caching issue on the server?

**Reply by joachimvh** [reply]
> @joachimvh could there be a caching issue on the server?

There was still a cache in the code, that might have been the cause so I just removed it completely so hopefully this issue is gone in the future.

https://linkedsoftwaredependencies.org/bundles/npm/@iotakingdoms/common is now reachable.

I do want the add a note for @ixuz that in general it should not be necessary for Components.js that this link works since you always want it to find the configurations locally.

**Reply by rubensworks** [reply]
Great, thanks @joachimvh!

**Reply by ixuz** [reply]
Thank you @rubensworks @joachimvh ! Great help as always! 👍
